scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -472,7 +473,8 @@ protected NodeSet getRMNodes(Map<JobId, JobDescriptor> jobMap, int neededResourc
      */
     private Map<String, Serializable> createBindingsForSelectionScripts(InternalJob job, EligibleTaskDescriptor etd, InternalTask task) throws IOException, ClassNotFoundException {
         Map<String, Serializable> bindings = new HashMap<>();
-        Map<String, Serializable> variables = new HashMap<>();
+        VariablesMap variables = new VariablesMap();
```

Comment:
Because I thought it would be better to always bind `SchedulerConstants.VARIABLES_BINDING_NAME` to an object of type `VariablesMap`, and not to an object whose type is different depending on the use